.github/workflows/build_and_test.yml

```diff
@@ -600,12 +602,12 @@ jobs:
     - uses: ilammy/msvc-dev-cmd@v1
     - name: install cxx bridge
       run: cargo install cxxbridge-cmd
-    - name: Build fuzzers/libfuzzer_stb_image 
+    - name: Build fuzzers/libfuzzer_stb_image
       run: cd fuzzers/libfuzzer_stb_image && cargo build --release
     - name: Build fuzzers/frida_libpng
       run: cd fuzzers/frida_libpng/ && cargo make test
     - name: Build fuzzers/frida_gdiplus
-      run: cd fuzzers/frida_gdiplus/ && cargo make test
+      run: cd fuzzers/frida_gdiplus/ && cargo make test && cargo make test_cmplog
```

Comment:
nice, CI is appreaciated! :)

Reply:
thanks, these asm tests is a must have :)